airflow/models/baseoperator.py

```diff
@@ -1204,7 +1204,7 @@ def clear(
             tasks += [t.task_id for t in self.get_flat_relatives(upstream=True)]
 
         if downstream:
-            tasks += [t.task_id for t in self.get_flat_relatives(upstream=False)]
+            tasks += [t.task_id for t in self.get_flat_relatives()]
```

Comment:
The default value of `upstream` is False, and we need to keep it for backward compatibility, but I’d prefer to keep it at the call sites for explicitness.